articles/quickstart/webapp/rails/01-login.md

```diff
@@ -47,15 +47,25 @@ If you are using Windows, uncomment the `tzinfo-data` gem in the Gemfile.
 
 ### Initialize OmniAuth Auth0
 
+Add your Auth0 configuration to the `secrets.yml` file for later use:
+
+```yaml
+development:
+  secret_key_base: d2b31d76ebec2f43b57eca7ee896e88b03177bddbb6511f7301c96e516ebf84a090520a8f6575e4047c693ebc6a96784afe18bc9b34b85827eb17b4ad0758639
```

Comment:
What is the purpose of this key? Where is it referenced and is it supposed to contain a value or a placeholder?

Reply:
Good point. This is for Rails to use as a [secret for cookie encryption amongst other things](https://api.rubyonrails.org/classes/Rails/Application.html#method-i-secret_key_base). It's there by default in a new project. Let me make it a placeholder instead.